manager/controllers/app/blueprint_controller.go

```diff
@@ -144,16 +148,20 @@ func getDomainFromImageName(image string) (string, error) {
 	return distributionref.Domain(named), nil
 }
 
-func (r *BlueprintReconciler) applyChartResource(ctx context.Context, log logr.Logger, chartSpec app.ChartSpec, args map[string]interface{}, blueprint *app.Blueprint, releaseName string) (ctrl.Result, error) {
-	log.Info(fmt.Sprintf("--- Chart Ref ---\n\n%v\n\n", chartSpec.Name))
+func (r *BlueprintReconciler) applyChartResource(ctx context.Context, log zerolog.Logger, chartSpec app.ChartSpec, args map[string]interface{}, blueprint *app.Blueprint, releaseName string) (ctrl.Result, error) {
+	// Get the unique id for the specific fybrikapplication instance.  Used for logging.
+	uuid := blueprint.GetAnnotations()[utils.FYBRIKAPPUUID]
```

Comment:
Sometimes a function from utils is used to extract UUID. Can this be made consistent over the code?

Reply:
Yes, changing